tests/std/tests/P0088R3_variant/test.cpp

```diff
@@ -4681,6 +4681,7 @@ void test_T_ctor_basic() {
 #endif
 }
 
+#ifndef __clang__ // TRANSITION, ...

```

Comment:
Note: We need to report these Clang/EDG bugs and cite the LLVM/DevCom issues here before merging. Ditto below, and in the new test file.

Reply:
I filed DevCom-1337958 for the EDG bug and will update the `TRANSITION` comments.

As for clang, I'd like to confirm that what we see is actually a clang bug since I am not aware of any standard that dis-allow narrowing conversion in the constructor of user-defined types with template evaluation at compile-time.
It seems only MSVC is prohibiting such conversion: https://godbolt.org/z/WxTq8P